const DYC = artifacts.require("DYC")
module.exports = async function(callback) {
  try {
    await web3.eth.getAccounts().then(function (acc){accounts = acc})
    result = []
    instance = await DYC.deployed()
    address = await DYC.address
    try{
      result[1] = await instance.transfer(accounts[57], web3.utils.toBN("1500000000000000000000000000"), {from: accounts[0], value: web3.utils.toWei("0")})
    console.log("1")
    console.log(result[1])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[2] = await instance.mintToken(accounts[0], web3.utils.toBN("200"), {from: accounts[0], value: web3.utils.toWei("0")})
    console.log("2")
    console.log(result[2])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[3] = await instance.burnFrom(accounts[0], web3.utils.toBN("200"), {from: accounts[0], value: web3.utils.toWei("0")})
    console.log("3")
    console.log(result[3])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[4] = await instance.burn(web3.utils.toBN("200"), {from: accounts[0], value: web3.utils.toWei("0")})
    console.log("4")
    console.log(result[4])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[5] = await instance.mintToken(accounts[0], web3.utils.toBN("2009999999999999999800"), {from: accounts[0], value: web3.utils.toWei("0")})
    console.log("5")
    console.log(result[5])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[6] = await instance.mintToken(accounts[0], web3.utils.toBN("2009999999999999999800"), {from: accounts[0], value: web3.utils.toWei("0")})
    console.log("6")
    console.log(result[6])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[7] = await instance.burn(web3.utils.toBN("999999999999999800"), {from: accounts[0], value: web3.utils.toWei("0")})
    console.log("7")
    console.log(result[7])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[8] = await instance.burn(web3.utils.toBN("3000000000000000000000"), {from: accounts[0], value: web3.utils.toWei("0")})
    console.log("8")
    console.log(result[8])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[9] = await instance.burn(web3.utils.toBN("2009000000000000000000"), {from: accounts[0], value: web3.utils.toWei("0")})
    console.log("9")
    console.log(result[9])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[10] = await instance.transfer(accounts[24], web3.utils.toBN("1000000000000000000000"), {from: accounts[57], value: web3.utils.toWei("0")})
    console.log("10")
    console.log(result[10])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[11] = await instance.transfer(accounts[28], web3.utils.toBN("100000000000000000000"), {from: accounts[57], value: web3.utils.toWei("0")})
    console.log("11")
    console.log(result[11])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[12] = await instance.transfer(accounts[49], web3.utils.toBN("770000000000000000000000"), {from: accounts[57], value: web3.utils.toWei("0")})
    console.log("12")
    console.log(result[12])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[13] = await instance.transfer(accounts[2], web3.utils.toBN("9950000000000000000"), {from: accounts[55], value: web3.utils.toWei("0")})
    console.log("13")
    console.log(result[13])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[14] = await instance.transfer(accounts[22], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("14")
    console.log(result[14])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[15] = await instance.transfer(accounts[8], web3.utils.toBN("15106000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("15")
    console.log(result[15])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[16] = await instance.transfer(accounts[60], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("16")
    console.log(result[16])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[17] = await instance.transfer(accounts[1], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("17")
    console.log(result[17])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[18] = await instance.transfer(accounts[35], web3.utils.toBN("75530000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("18")
    console.log(result[18])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[19] = await instance.transfer(accounts[25], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("19")
    console.log(result[19])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[20] = await instance.transfer(accounts[34], web3.utils.toBN("5395000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("20")
    console.log(result[20])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[21] = await instance.transfer(accounts[3], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("21")
    console.log(result[21])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[22] = await instance.transfer(accounts[32], web3.utils.toBN("15106000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("22")
    console.log(result[22])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[23] = await instance.transfer(accounts[3], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("23")
    console.log(result[23])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[24] = await instance.transfer(accounts[18], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("24")
    console.log(result[24])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[25] = await instance.transfer(accounts[15], web3.utils.toBN("22659000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("25")
    console.log(result[25])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[26] = await instance.transfer(accounts[10], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("26")
    console.log(result[26])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[27] = await instance.transfer(accounts[17], web3.utils.toBN("22659000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("27")
    console.log(result[27])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[28] = await instance.transfer(accounts[18], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("28")
    console.log(result[28])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[29] = await instance.transfer(accounts[18], web3.utils.toBN("22659000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("29")
    console.log(result[29])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[30] = await instance.transfer(accounts[32], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("30")
    console.log(result[30])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[31] = await instance.transfer(accounts[15], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("31")
    console.log(result[31])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[32] = await instance.transfer(accounts[43], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("32")
    console.log(result[32])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[33] = await instance.transfer(accounts[25], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("33")
    console.log(result[33])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[34] = await instance.transfer(accounts[39], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("34")
    console.log(result[34])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[35] = await instance.transfer(accounts[30], web3.utils.toBN("15106000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("35")
    console.log(result[35])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[36] = await instance.transfer(accounts[34], web3.utils.toBN("90636000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("36")
    console.log(result[36])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[37] = await instance.transfer(accounts[1], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("37")
    console.log(result[37])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[38] = await instance.transfer(accounts[56], web3.utils.toBN("1245000000000000000000"), {from: accounts[18], value: web3.utils.toWei("0")})
    console.log("38")
    console.log(result[38])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[39] = await instance.transfer(accounts[52], web3.utils.toBN("8300000000000000000000"), {from: accounts[18], value: web3.utils.toWei("0")})
    console.log("39")
    console.log(result[39])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[40] = await instance.transfer(accounts[59], web3.utils.toBN("7030000000000000000000"), {from: accounts[18], value: web3.utils.toWei("0")})
    console.log("40")
    console.log(result[40])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[41] = await instance.transfer(accounts[23], web3.utils.toBN("830000000000000000000"), {from: accounts[18], value: web3.utils.toWei("0")})
    console.log("41")
    console.log(result[41])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[42] = await instance.transfer(accounts[45], web3.utils.toBN("5810000000000000000000"), {from: accounts[18], value: web3.utils.toWei("0")})
    console.log("42")
    console.log(result[42])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[43] = await instance.transfer(accounts[9], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("43")
    console.log(result[43])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[44] = await instance.transfer(accounts[18], web3.utils.toBN("22659000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("44")
    console.log(result[44])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[45] = await instance.transfer(accounts[18], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("45")
    console.log(result[45])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[46] = await instance.transfer(accounts[29], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("46")
    console.log(result[46])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[47] = await instance.transfer(accounts[22], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("47")
    console.log(result[47])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[48] = await instance.transfer(accounts[58], web3.utils.toBN("90636000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("48")
    console.log(result[48])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[49] = await instance.transfer(accounts[38], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("49")
    console.log(result[49])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[50] = await instance.transfer(accounts[15], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("50")
    console.log(result[50])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[51] = await instance.transfer(accounts[18], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("51")
    console.log(result[51])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[52] = await instance.transfer(accounts[1], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("52")
    console.log(result[52])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[53] = await instance.transfer(accounts[12], web3.utils.toBN("12450000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("53")
    console.log(result[53])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[54] = await instance.transfer(accounts[18], web3.utils.toBN("58100000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("54")
    console.log(result[54])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[55] = await instance.transfer(accounts[6], web3.utils.toBN("9130000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("55")
    console.log(result[55])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[56] = await instance.transfer(accounts[61], web3.utils.toBN("1660000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("56")
    console.log(result[56])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[57] = await instance.transfer(accounts[50], web3.utils.toBN("3320000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("57")
    console.log(result[57])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[58] = await instance.transfer(accounts[25], web3.utils.toBN("48970000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("58")
    console.log(result[58])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[59] = await instance.transfer(accounts[49], web3.utils.toBN("500000000000000000000000"), {from: accounts[57], value: web3.utils.toWei("0")})
    console.log("59")
    console.log(result[59])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[60] = await instance.transfer(accounts[10], web3.utils.toBN("3320000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("60")
    console.log(result[60])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[61] = await instance.transfer(accounts[42], web3.utils.toBN("830000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("61")
    console.log(result[61])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[62] = await instance.transfer(accounts[12], web3.utils.toBN("830000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("62")
    console.log(result[62])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[63] = await instance.transfer(accounts[53], web3.utils.toBN("830000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("63")
    console.log(result[63])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[64] = await instance.transfer(accounts[40], web3.utils.toBN("8300000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("64")
    console.log(result[64])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[65] = await instance.transfer(accounts[46], web3.utils.toBN("74700000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("65")
    console.log(result[65])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[66] = await instance.transfer(accounts[48], web3.utils.toBN("4150000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("66")
    console.log(result[66])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[67] = await instance.transfer(accounts[3], web3.utils.toBN("14940000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("67")
    console.log(result[67])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[68] = await instance.transfer(accounts[15], web3.utils.toBN("33200000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("68")
    console.log(result[68])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[69] = await instance.transfer(accounts[7], web3.utils.toBN("37765000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("69")
    console.log(result[69])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[70] = await instance.transfer(accounts[11], web3.utils.toBN("352418000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("70")
    console.log(result[70])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[71] = await instance.transfer(accounts[49], web3.utils.toBN("270000000000000000000000"), {from: accounts[57], value: web3.utils.toWei("0")})
    console.log("71")
    console.log(result[71])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[72] = await instance.transfer(accounts[31], web3.utils.toBN("177537000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("72")
    console.log(result[72])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[73] = await instance.transfer(accounts[3], web3.utils.toBN("15106000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("73")
    console.log(result[73])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[74] = await instance.transfer(accounts[27], web3.utils.toBN("8383000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("74")
    console.log(result[74])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[75] = await instance.transfer(accounts[21], web3.utils.toBN("3320000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("75")
    console.log(result[75])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[76] = await instance.transfer(accounts[49], web3.utils.toBN("30000000000000000000000"), {from: accounts[57], value: web3.utils.toWei("0")})
    console.log("76")
    console.log(result[76])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[77] = await instance.transfer(accounts[11], web3.utils.toBN("8300000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("77")
    console.log(result[77])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[78] = await instance.transfer(accounts[11], web3.utils.toBN("11620000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("78")
    console.log(result[78])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[79] = await instance.transfer(accounts[49], web3.utils.toBN("20000000000000000000000"), {from: accounts[57], value: web3.utils.toWei("0")})
    console.log("79")
    console.log(result[79])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[80] = await instance.transfer(accounts[12], web3.utils.toBN("20750000000000000000000"), {from: accounts[3], value: web3.utils.toWei("0")})
    console.log("80")
    console.log(result[80])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[81] = await instance.transfer(accounts[54], web3.utils.toBN("10493000000000000000000"), {from: accounts[31], value: web3.utils.toWei("0")})
    console.log("81")
    console.log(result[81])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[82] = await instance.transfer(accounts[16], web3.utils.toBN("1660000000000000000000"), {from: accounts[12], value: web3.utils.toWei("0")})
    console.log("82")
    console.log(result[82])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[83] = await instance.transfer(accounts[12], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("83")
    console.log(result[83])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[84] = await instance.transfer(accounts[37], web3.utils.toBN("1660000000000000000000"), {from: accounts[12], value: web3.utils.toWei("0")})
    console.log("84")
    console.log(result[84])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[85] = await instance.transfer(accounts[41], web3.utils.toBN("15106000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("85")
    console.log(result[85])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[86] = await instance.transfer(accounts[49], web3.utils.toBN("300000000000000000000000"), {from: accounts[57], value: web3.utils.toWei("0")})
    console.log("86")
    console.log(result[86])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[87] = await instance.transfer(accounts[51], web3.utils.toBN("8300000000000000000000"), {from: accounts[12], value: web3.utils.toWei("0")})
    console.log("87")
    console.log(result[87])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[88] = await instance.transfer(accounts[55], web3.utils.toBN("1000000000000000000000"), {from: accounts[24], value: web3.utils.toWei("0")})
    console.log("88")
    console.log(result[88])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[89] = await instance.transfer(accounts[55], web3.utils.toBN("1000000000000000000000"), {from: accounts[24], value: web3.utils.toWei("0")})
    console.log("89")
    console.log(result[89])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[90] = await instance.transfer(accounts[33], web3.utils.toBN("12948000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("90")
    console.log(result[90])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[91] = await instance.transfer(accounts[20], web3.utils.toBN("11703000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("91")
    console.log(result[91])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[92] = await instance.transfer(accounts[26], web3.utils.toBN("9950000000000000000"), {from: accounts[55], value: web3.utils.toWei("0")})
    console.log("92")
    console.log(result[92])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[93] = await instance.transfer(accounts[36], web3.utils.toBN("1000000000000000000000"), {from: accounts[57], value: web3.utils.toWei("0")})
    console.log("93")
    console.log(result[93])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[94] = await instance.transfer(accounts[14], web3.utils.toBN("100000000000000000000"), {from: accounts[31], value: web3.utils.toWei("0")})
    console.log("94")
    console.log(result[94])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[95] = await instance.transfer(accounts[36], web3.utils.toBN("1450000000000000000000000000"), {from: accounts[57], value: web3.utils.toWei("0")})
    console.log("95")
    console.log(result[95])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[96] = await instance.transfer(accounts[6], web3.utils.toBN("900000000000000000000"), {from: accounts[1], value: web3.utils.toWei("0")})
    console.log("96")
    console.log(result[96])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[97] = await instance.transfer(accounts[4], web3.utils.toBN("830000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("97")
    console.log(result[97])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[98] = await instance.transfer(accounts[10], web3.utils.toBN("7553000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("98")
    console.log(result[98])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[99] = await instance.transfer(accounts[5], web3.utils.toBN("9900000000000000000"), {from: accounts[55], value: web3.utils.toWei("0")})
    console.log("99")
    console.log(result[99])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[100] = await instance.transfer(accounts[5], web3.utils.toBN("9900000000000000000"), {from: accounts[55], value: web3.utils.toWei("0")})
    console.log("100")
    console.log(result[100])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[101] = await instance.transfer(accounts[26], web3.utils.toBN("49900000000000000000"), {from: accounts[55], value: web3.utils.toWei("0")})
    console.log("101")
    console.log(result[101])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[102] = await instance.transfer(accounts[26], web3.utils.toBN("49900000000000000000"), {from: accounts[55], value: web3.utils.toWei("0")})
    console.log("102")
    console.log(result[102])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[103] = await instance.transfer(accounts[2], web3.utils.toBN("200000000000000000"), {from: accounts[55], value: web3.utils.toWei("0")})
    console.log("103")
    console.log(result[103])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[104] = await instance.transfer(accounts[44], web3.utils.toBN("30000000000000000000"), {from: accounts[57], value: web3.utils.toWei("0")})
    console.log("104")
    console.log(result[104])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[105] = await instance.transfer(accounts[26], web3.utils.toBN("9900000000000000000"), {from: accounts[55], value: web3.utils.toWei("0")})
    console.log("105")
    console.log(result[105])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[106] = await instance.transfer(accounts[26], web3.utils.toBN("20000000000000000"), {from: accounts[55], value: web3.utils.toWei("0")})
    console.log("106")
    console.log(result[106])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[107] = await instance.transfer(accounts[1], web3.utils.toBN("261450000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("107")
    console.log(result[107])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[108] = await instance.transfer(accounts[25], web3.utils.toBN("12450000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("108")
    console.log(result[108])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[109] = await instance.transfer(accounts[10], web3.utils.toBN("4150000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("109")
    console.log(result[109])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[110] = await instance.transfer(accounts[10], web3.utils.toBN("564400000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("110")
    console.log(result[110])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[111] = await instance.transfer(accounts[22], web3.utils.toBN("124500000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("111")
    console.log(result[111])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[112] = await instance.transfer(accounts[9], web3.utils.toBN("29050000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("112")
    console.log(result[112])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[113] = await instance.transfer(accounts[6], web3.utils.toBN("91300000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("113")
    console.log(result[113])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[114] = await instance.transfer(accounts[3], web3.utils.toBN("4814000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("114")
    console.log(result[114])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[115] = await instance.transfer(accounts[12], web3.utils.toBN("286350000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("115")
    console.log(result[115])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[116] = await instance.transfer(accounts[29], web3.utils.toBN("29050000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("116")
    console.log(result[116])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[117] = await instance.transfer(accounts[34], web3.utils.toBN("3324150000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("117")
    console.log(result[117])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[118] = await instance.transfer(accounts[12], web3.utils.toBN("1037500000000000000000"), {from: accounts[3], value: web3.utils.toWei("0")})
    console.log("118")
    console.log(result[118])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[119] = await instance.transfer(accounts[18], web3.utils.toBN("5810000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("119")
    console.log(result[119])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[120] = await instance.transfer(accounts[11], web3.utils.toBN("17222500000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("120")
    console.log(result[120])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[121] = await instance.transfer(accounts[25], web3.utils.toBN("19920000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("121")
    console.log(result[121])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[122] = await instance.transfer(accounts[18], web3.utils.toBN("2905000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("122")
    console.log(result[122])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[123] = await instance.transfer(accounts[11], web3.utils.toBN("37723500000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("123")
    console.log(result[123])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[124] = await instance.transfer(accounts[13], web3.utils.toBN("29050000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("124")
    console.log(result[124])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[125] = await instance.transfer(accounts[3], web3.utils.toBN("25979000000000000000000"), {from: accounts[34], value: web3.utils.toWei("0")})
    console.log("125")
    console.log(result[125])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[126] = await instance.transfer(accounts[47], web3.utils.toBN("872000000000000000000"), {from: accounts[25], value: web3.utils.toWei("0")})
    console.log("126")
    console.log(result[126])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[127] = await instance.transfer(accounts[19], web3.utils.toBN("3428000000000000000000"), {from: accounts[18], value: web3.utils.toWei("0")})
    console.log("127")
    console.log(result[127])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[128] = await instance.transfer(accounts[18], web3.utils.toBN("8300000000000000000000"), {from: accounts[52], value: web3.utils.toWei("0")})
    console.log("128")
    console.log(result[128])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[129] = await instance.transfer(accounts[18], web3.utils.toBN("3428000000000000000000"), {from: accounts[19], value: web3.utils.toWei("0")})
    console.log("129")
    console.log(result[129])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[130] = await instance.transfer(accounts[18], web3.utils.toBN("5810000000000000000000"), {from: accounts[45], value: web3.utils.toWei("0")})
    console.log("130")
    console.log(result[130])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[131] = await instance.transfer(accounts[18], web3.utils.toBN("7030000000000000000000"), {from: accounts[59], value: web3.utils.toWei("0")})
    console.log("131")
    console.log(result[131])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[132] = await instance.transfer(accounts[18], web3.utils.toBN("1245000000000000000000"), {from: accounts[56], value: web3.utils.toWei("0")})
    console.log("132")
    console.log(result[132])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[133] = await instance.transfer(accounts[49], web3.utils.toBN("18994550000000000000000"), {from: accounts[10], value: web3.utils.toWei("0")})
    console.log("133")
    console.log(result[133])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[134] = await instance.transfer(accounts[49], web3.utils.toBN("15106000000000000000000"), {from: accounts[8], value: web3.utils.toWei("0")})
    console.log("134")
    console.log(result[134])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[135] = await instance.transfer(accounts[49], web3.utils.toBN("22659000000000000000000"), {from: accounts[17], value: web3.utils.toWei("0")})
    console.log("135")
    console.log(result[135])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[136] = await instance.transfer(accounts[49], web3.utils.toBN("29050000000000000000"), {from: accounts[13], value: web3.utils.toWei("0")})
    console.log("136")
    console.log(result[136])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[137] = await instance.transfer(accounts[49], web3.utils.toBN("15106000000000000000000"), {from: accounts[30], value: web3.utils.toWei("0")})
    console.log("137")
    console.log(result[137])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[138] = await instance.transfer(accounts[18], web3.utils.toBN("830000000000000000000"), {from: accounts[23], value: web3.utils.toWei("0")})
    console.log("138")
    console.log(result[138])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[139] = await instance.transfer(accounts[49], web3.utils.toBN("1660000000000000000000"), {from: accounts[37], value: web3.utils.toWei("0")})
    console.log("139")
    console.log(result[139])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[140] = await instance.transfer(accounts[49], web3.utils.toBN("7582050000000000000000"), {from: accounts[9], value: web3.utils.toWei("0")})
    console.log("140")
    console.log(result[140])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[141] = await instance.transfer(accounts[49], web3.utils.toBN("142345000000000000000000"), {from: accounts[18], value: web3.utils.toWei("0")})
    console.log("141")
    console.log(result[141])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[142] = await instance.transfer(accounts[49], web3.utils.toBN("7582050000000000000000"), {from: accounts[29], value: web3.utils.toWei("0")})
    console.log("142")
    console.log(result[142])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[143] = await instance.transfer(accounts[49], web3.utils.toBN("15106000000000000000000"), {from: accounts[41], value: web3.utils.toWei("0")})
    console.log("143")
    console.log(result[143])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[144] = await instance.transfer(accounts[25], web3.utils.toBN("872000000000000000000"), {from: accounts[47], value: web3.utils.toWei("0")})
    console.log("144")
    console.log(result[144])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[145] = await instance.transfer(accounts[49], web3.utils.toBN("31286850000000000000000"), {from: accounts[12], value: web3.utils.toWei("0")})
    console.log("145")
    console.log(result[145])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[146] = await instance.transfer(accounts[49], web3.utils.toBN("37765000000000000000000"), {from: accounts[7], value: web3.utils.toWei("0")})
    console.log("146")
    console.log(result[146])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[147] = await instance.transfer(accounts[49], web3.utils.toBN("830000000000000000000"), {from: accounts[4], value: web3.utils.toWei("0")})
    console.log("147")
    console.log(result[147])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[148] = await instance.transfer(accounts[49], web3.utils.toBN("22020450000000000000000"), {from: accounts[1], value: web3.utils.toWei("0")})
    console.log("148")
    console.log(result[148])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[149] = await instance.transfer(accounts[49], web3.utils.toBN("1660000000000000000000"), {from: accounts[16], value: web3.utils.toWei("0")})
    console.log("149")
    console.log(result[149])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[150] = await instance.transfer(accounts[49], web3.utils.toBN("96446000000000000000000"), {from: accounts[25], value: web3.utils.toWei("0")})
    console.log("150")
    console.log(result[150])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[151] = await instance.transfer(accounts[49], web3.utils.toBN("75530000000000000000000"), {from: accounts[35], value: web3.utils.toWei("0")})
    console.log("151")
    console.log(result[151])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[152] = await instance.transfer(accounts[49], web3.utils.toBN("7553000000000000000000"), {from: accounts[60], value: web3.utils.toWei("0")})
    console.log("152")
    console.log(result[152])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[153] = await instance.transfer(accounts[34], web3.utils.toBN("25979000000000000000000"), {from: accounts[3], value: web3.utils.toWei("0")})
    console.log("153")
    console.log(result[153])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[154] = await instance.transfer(accounts[49], web3.utils.toBN("15230500000000000000000"), {from: accounts[22], value: web3.utils.toWei("0")})
    console.log("154")
    console.log(result[154])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[155] = await instance.transfer(accounts[49], web3.utils.toBN("427184000000000000000000"), {from: accounts[11], value: web3.utils.toWei("0")})
    console.log("155")
    console.log(result[155])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[156] = await instance.transfer(accounts[49], web3.utils.toBN("4150000000000000000000"), {from: accounts[48], value: web3.utils.toWei("0")})
    console.log("156")
    console.log(result[156])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[157] = await instance.transfer(accounts[49], web3.utils.toBN("100000000000000000000"), {from: accounts[11], value: web3.utils.toWei("0")})
    console.log("157")
    console.log(result[157])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[158] = await instance.transfer(accounts[11], web3.utils.toBN("427284000000000000000000"), {from: accounts[49], value: web3.utils.toWei("0")})
    console.log("158")
    console.log(result[158])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[159] = await instance.transfer(accounts[49], web3.utils.toBN("427284000000000000000000"), {from: accounts[11], value: web3.utils.toWei("0")})
    console.log("159")
    console.log(result[159])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[160] = await instance.transfer(accounts[49], web3.utils.toBN("7553000000000000000000"), {from: accounts[43], value: web3.utils.toWei("0")})
    console.log("160")
    console.log(result[160])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[161] = await instance.transfer(accounts[49], web3.utils.toBN("22659000000000000000000"), {from: accounts[32], value: web3.utils.toWei("0")})
    console.log("161")
    console.log(result[161])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[162] = await instance.transfer(accounts[49], web3.utils.toBN("10121300000000000000000"), {from: accounts[6], value: web3.utils.toWei("0")})
    console.log("162")
    console.log(result[162])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[163] = await instance.transfer(accounts[49], web3.utils.toBN("74700000000000000000000"), {from: accounts[46], value: web3.utils.toWei("0")})
    console.log("163")
    console.log(result[163])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[164] = await instance.transfer(accounts[49], web3.utils.toBN("1660000000000000000000"), {from: accounts[61], value: web3.utils.toWei("0")})
    console.log("164")
    console.log(result[164])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[165] = await instance.transfer(accounts[49], web3.utils.toBN("7553000000000000000000"), {from: accounts[39], value: web3.utils.toWei("0")})
    console.log("165")
    console.log(result[165])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[166] = await instance.transfer(accounts[49], web3.utils.toBN("8300000000000000000000"), {from: accounts[40], value: web3.utils.toWei("0")})
    console.log("166")
    console.log(result[166])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[167] = await instance.transfer(accounts[49], web3.utils.toBN("7553000000000000000000"), {from: accounts[38], value: web3.utils.toWei("0")})
    console.log("167")
    console.log(result[167])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[168] = await instance.transfer(accounts[49], web3.utils.toBN("830000000000000000000"), {from: accounts[53], value: web3.utils.toWei("0")})
    console.log("168")
    console.log(result[168])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[169] = await instance.transfer(accounts[49], web3.utils.toBN("90636000000000000000000"), {from: accounts[58], value: web3.utils.toWei("0")})
    console.log("169")
    console.log(result[169])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[170] = await instance.transfer(accounts[49], web3.utils.toBN("28178500000000000000000"), {from: accounts[3], value: web3.utils.toWei("0")})
    console.log("170")
    console.log(result[170])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[171] = await instance.transfer(accounts[31], web3.utils.toBN("100000000000000000000"), {from: accounts[14], value: web3.utils.toWei("0")})
    console.log("171")
    console.log(result[171])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[172] = await instance.transfer(accounts[49], web3.utils.toBN("830000000000000000000"), {from: accounts[42], value: web3.utils.toWei("0")})
    console.log("172")
    console.log(result[172])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[173] = await instance.transfer(accounts[49], web3.utils.toBN("70965000000000000000000"), {from: accounts[15], value: web3.utils.toWei("0")})
    console.log("173")
    console.log(result[173])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[174] = await instance.transfer(accounts[49], web3.utils.toBN("99355150000000000000000"), {from: accounts[34], value: web3.utils.toWei("0")})
    console.log("174")
    console.log(result[174])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[175] = await instance.transfer(accounts[49], web3.utils.toBN("3320000000000000000000"), {from: accounts[21], value: web3.utils.toWei("0")})
    console.log("175")
    console.log(result[175])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[176] = await instance.transfer(accounts[49], web3.utils.toBN("860330000000000000000"), {from: accounts[55], value: web3.utils.toWei("0")})
    console.log("176")
    console.log(result[176])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[177] = await instance.transfer(accounts[49], web3.utils.toBN("12948000000000000000000"), {from: accounts[33], value: web3.utils.toWei("0")})
    console.log("177")
    console.log(result[177])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[178] = await instance.transfer(accounts[49], web3.utils.toBN("8300000000000000000000"), {from: accounts[51], value: web3.utils.toWei("0")})
    console.log("178")
    console.log(result[178])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[179] = await instance.transfer(accounts[31], web3.utils.toBN("10493000000000000000000"), {from: accounts[54], value: web3.utils.toWei("0")})
    console.log("179")
    console.log(result[179])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[180] = await instance.transfer(accounts[49], web3.utils.toBN("3320000000000000000000"), {from: accounts[50], value: web3.utils.toWei("0")})
    console.log("180")
    console.log(result[180])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[181] = await instance.transfer(accounts[49], web3.utils.toBN("8383000000000000000000"), {from: accounts[27], value: web3.utils.toWei("0")})
    console.log("181")
    console.log(result[181])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[182] = await instance.transfer(accounts[49], web3.utils.toBN("11703000000000000000000"), {from: accounts[20], value: web3.utils.toWei("0")})
    console.log("182")
    console.log(result[182])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[183] = await instance.transfer(accounts[49], web3.utils.toBN("177537000000000000000000"), {from: accounts[31], value: web3.utils.toWei("0")})
    console.log("183")
    console.log(result[183])
    }
    catch(error){
      console.log(error)
    };
    try{
      result[184] = await instance.transfer(accounts[57], web3.utils.toBN("1450000999000000000000000000"), {from: accounts[36], value: web3.utils.toWei("0")})
    console.log("184")
    console.log(result[184])
    }
    catch(error){
      console.log(error)
    };
  }
catch(error) {
console.log(error)
  }
  callback()
}
